envoy/api/v2/auth/local_rbac.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+// [#proto-status: draft]
+
+package envoy.api.v2.auth;
+
+import "envoy/api/v2/auth/rbac.proto";
+
+// LocalRBAC specifies Istio RBAC policies that apply to the local RBAC engine
```

Comment:
Do we want to keep the Istio references throughout, or just link back to the existing Istio documentation for reference?